packages/examples/wrapper_langium.html

```diff
@@ -11,7 +11,7 @@
     <h2>Langium Grammar DSL Language Client & Server (Worker)</h2>
     <div style="padding: 0px 5px 5px 0px">
         <button type="button" id="button-start-classic">Start Classic</button>
-        <button type="button" id="button-start-vscode-api">Start VscodeApi</button>
+        <button type="button" id="button-start-extended">Start Exentended</button>
```

Comment:
Another typo here: `Start Extended`

Reply:
Thanks for catching that. Will merge this after correction.